mdflynn/game-sleuth
# Feature/search form testing

## Is this a fix or a feature?
This is a feature

## What is the change?
- Add styling to the SearchForm component
- Unit test the SearchForm component
- 
## What does it fix?
No fix this PR

## Where should the reviewer start?
Review the entire SearchForm.tsx and SearchForm.test.tsx file

## How should this be tested?
Run `npm start` and navigate to localhost:3000 to see site styling
Run `npm test` to see passing test suite

## Relevant Issues
#22
